extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnPageReaderImpl.java

```diff
@@ -95,8 +90,10 @@ final class ColumnPageReaderImpl implements ColumnPageReader {
         this.path = path;
         this.uri = uri;
         this.fieldTypes = fieldTypes;
-        this.offset = offset;
+        this.dataOffset = dataOffset;
```

Comment:
Now we always get the offset of the data in the page, page header and number of values at the time of construction.